API-Reference/Macros.md

```diff
@@ -0,0 +1,132 @@
+| `KOKKOS_ARCH_KNC` | Optimize for Intel Knights Corner Xeon Phi (HOST) |
+| `KOKKOS_ARCH_AVX512MIC` | Optimize for Many Integrated Core (MIC; AVX512) |
+| `KOKKOS_ARCH_POWER7` | Optimize for IBM POWER7 CPUs (HOST) |
+| `KOKKOS_ARCH_POWER8` | Optimize for IBM POWER8 CPUs (HOST) |
+| `KOKKOS_ARCH_POWER9` | Optimize for IBM POWER9 CPUs (HOST)|
+| `KOKKOS_ARCH_INTEL_GEN` | Optimize for Intel GPUs Gen9+ (GPU) |
+| `KOKKOS_ARCH_INTEL_DG1` | Optimize for Intel Iris XeMAX GPU (GPU) |
+| `KOKKOS_ARCH_INTEL_GEN9` | Optimize for Intel GPU Gen9 (GPU)|
+| `KOKKOS_ARCH_INTEL_GEN11` | Optimize for Intel GPU Gen11 (GPU) |
+| `KOKKOS_ARCH_INTEL_GEN12LP` | Optimize for Intel GPU Gen12LP (GPU) |
+| `KOKKOS_ARCH_INTEL_XEHP` | Optimize for Intel GPU Xe-HP (GPU) |
+| `KOKKOS_ARCH_INTEL_GPU` | Set if any Intel GPU architecture has been enabled |
+| `KOKKOS_ARCH_KEPLER` | Set if any NVIDIA Kepler architecture has been enabled |
+| `KOKKOS_ARCH_KEPLER30` | Optimize for NVIDIA Kepler generation CC 3.0 (GPU) |
+| `KOKKOS_ARCH_KEPLER32` | Optimize for NVIDIA Kepler generation CC 3.2 (GPU) |
+| `KOKKOS_ARCH_KEPLER35` | Optimize for NVIDIA Kepler generation CC 3.5 (GPU) |
+| `KOKKOS_ARCH_KEPLER37` | Optimize for NVIDIA Kepler generation CC 3.7 (GPU) |
+| `KOKKOS_ARCH_MAXWELL` | Set if any NVIDIA Maxwell architecture has been enabled |
+| `KOKKOS_ARCH_MAXWELL50` | Optimize for NVIDIA Maxwell generation CC 5.0 (GPU) |
+| `KOKKOS_ARCH_MAXWELL52` | Optimize for NVIDIA Maxwell generation CC 5.2 (GPU) |
+| `KOKKOS_ARCH_MAXWELL53` | Optimize for NVIDIA Maxwell generation CC 5.3 (GPU) |
+| `KOKKOS_ARCH_PASCAL` | Set if any NVIDIA Pascal architecture has been enabled  |
+| `KOKKOS_ARCH_PASCAL60` | Optimize for NVIDIA Pascal generation CC 6.0 (GPU) |
+| `KOKKOS_ARCH_PASCAL61` | Optimize for NVIDIA Pascal generation CC 6.1 (GPU) |
+| `KOKKOS_ARCH_VOLTA` | Set if any NVIDIA Volta architecture has been enabled |
+| `KOKKOS_ARCH_VOLTA70` | Optimize for NVIDIA Volta generation CC 7.0 (GPU) |
+| `KOKKOS_ARCH_VOLTA72` | Optimize for NVIDIA Volta generation CC 7.2 (GPU) |
+| `KOKKOS_ARCH_TURING75` | Optimize for NVIDIA Turing generation CC 7.5 (GPU) |
+| `KOKKOS_ARCH_AMPERE` | Set if any NVIDIA Ampere architecture has been enabled |
+| `KOKKOS_ARCH_AMPERE80` | Optimize for NVIDIA Ampere generation CC 8.0 (GPU) |
+| `KOKKOS_ARCH_AMPERE86 ` | Optimize for NVIDIA Ampere generation CC 8.6 (GPU) |
+| `KOKKOS_ARCH_AMD_ZEN` | Optimize for AMD Zen architecture (HOST) |
+| `KOKKOS_ARCH_AMD_ZEN2` | Optimize for AMD Zen2 architecture (HOST) |
+| `KOKKOS_ARCH_AMD_ZEN3` | Optimize for AMD Zen3 architecture (HOST) |
+| `KOKKOS_ARCH_VEGA` | Set if any AMD Vega GPU architecture as been enabled |
+| `KOKKOS_ARCH_VEGA900` | Optimize for AMD GPU MI25 GFX900 (GPU) |
+| `KOKKOS_ARCH_VEGA906` | Optimize for AMD GPU MI50/MI60 GFX906 (GPU) |
+| `KOKKOS_ARCH_VEGA908` | Optimize for AMD GPU MI100 GFX908 (GPU) |
+| `KOKKOS_ARCH_VEGA90A` | Optimize for AMD GPU MI200 GFX90A (GPU) |
```

Comment:
@Rombur should we say "MI200 series" to encompass MI210, MI250, and MI250X

Reply:
yes, we can change it to series